internal/k6runner/k6runner_test.go

```diff
@@ -178,6 +179,11 @@ func TestScriptHTTPRun(t *testing.T) {
 	var (
 		testMetrics = testhelper.MustReadFile(t, "testdata/test.out")
 		testLogs    = testhelper.MustReadFile(t, "testdata/test.log")
+		// nonDebugLogLine is a log line with level != debug present in testLogs.
+		// Non-debug lines are the ones that are actually submitted to the logger, thus the ones we expect to find on it
+		// during testing.
```

Comment:
nit: Maybe is because I lack some context with the runner, but I find this explanation a little bit confusing. It mentions `debug` level, but it is not clear from here the reasoning for that, unless you go search the implementation and see that we are explicitly discarding any log line with `debug` level and no `source` label.

Reply:
You're most likely right, it was not very obvious why I was looking specifically for this one. I've reworded this a bit, LMK what you think!